### ISSUE TITLE
Focus is not shown the first time you tab into the table in Firefox

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#261]
When tabbing into the table in firefox you can not se where the focus in the table is because no focus is displayed. If you tab a second time then the focus is displayed.

**Comment by haxxmaxx** [comment]
This might be solved with the new nebula update
